sdf_timing/sdfwrite.py

```diff
@@ -233,7 +233,7 @@ def emit_sdf(timings, timescale='1ps'):
                 for location in sorted(timings['cells'][cell]):
                     sdf += """
     (CELL
-        (CELLTYPE \"{name}\")""".format(name=cell.upper())
+        (CELLTYPE \"{name}\")""".format(name=cell)
```

Comment:
Why are you removing the `upper()`?

Reply:
According to the SDF specification the CELLTYPE does not have to be uppercase. And before that change the round trip SDF -> python_sdf_parse -> SDF caused all these to be made uppercase.

Reply:
It would probably good to have a "uppercase everything" mode?

Reply:
That makes sense. I've added an additional parameter to the `emit_sdf` function.